src/Payum/Stripe/Resources/views/Action/obtain_js_token.html.twig

```diff
@@ -45,26 +45,6 @@
     <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
could you create a separate block payum_vendor_javascripts and move these to it.